frontend/app/module/input/PollingStation/AbortDataEntry.test.tsx

```diff
@@ -0,0 +1,73 @@
+import * as router from "react-router";
+
+import { afterEach, beforeEach, describe, expect, test, vi } from "vitest";
+
+import { fireEvent, overrideOnce, render, screen, server, waitFor } from "app/test/unit";
+
+import { ElectionProvider, PollingStationFormController } from "@kiesraad/api";
+import { electionMock, electionMockResponse, pollingStationMock } from "@kiesraad/api-mocks";
+
+import { AbortDataEntry } from "./AbortDataEntry.tsx";
+
+const mockNavigate = vi.fn();
+
+const renderAbortDataEntry = () => {
+  render(
+    <ElectionProvider electionId={1}>
+      <PollingStationFormController
+        election={electionMock}
+        pollingStationId={pollingStationMock.id}
+        entryNumber={1}
+      >
+        <AbortDataEntry />
+      </PollingStationFormController>
+    </ElectionProvider>,
+  );
+};
+
+describe("Test AbortDataEntry", () => {
+  beforeEach(async () => {
+    overrideOnce("get", "/api/elections/1", 200, electionMockResponse);
+    vi.spyOn(router, "useNavigate").mockImplementation(() => mockNavigate);
+
+    // render the abort button, then click it to open the modal
+    renderAbortDataEntry();
+    const abortButton = await screen.findByText("Invoer afbreken");
+    fireEvent.click(abortButton);
```

Comment:
```suggestion
    const abortButton = await screen.findByRole("button", { name: "Invoer afbreken"});
    const user = userEvent.setup();
    user.click(abortButton);
```

Reply:
`findByRole()` because it's more specific.
`userEvent` because it's closer to what users do: https://testing-library.com/docs/user-event/intro/

Reply:
Changed in 56c6749.